corehq/apps/users/management/commands/add_program_manager_role.py

```diff
@@ -0,0 +1,41 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.models_role import UserRole
+from corehq.apps.users.role_utils import UserRolePresets
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.accounting.models import SoftwarePlanEdition
+
+
+class Command(BaseCommand):
```

Comment:
Dear reviewer, should I add an aurguement here for us to use in order to remove all `UserRole` record where the name is `Program Manager`? This would only be used if things goes south and we have to revert.